packages/devtools_app/benchmark/README.md

```diff
@@ -23,6 +23,16 @@ To run the performance benchmark tests locally, run:
 dart run benchmark/scripts/run_benchmarks.dart
 ```
 
+Several arguments are available from the `run_benchmarks.dart` script. Using
+these arguments, you can do things like:
+* compute the average of multiple benchmark runs
```

Comment:
How about making it shorter?

Provide parameters for `run_benchmarks.dart` in order to: